### DISCUSSION TITLE
Client Portal Mobile App

### DISCUSSION THREAD

**Discussion by pranali98-cpu** [madrecha/jamku#951] [Announcements]
Recently we have are seeing uptick in usage of client portal. Many offices have requested us for super easy and user-friendly mobile app for client portal. 
In last three weeks we have been designing mobile app which would provide a native mobile app experience and at the same time provide most relevant details to client. 
~We expect to launch this mobile app in next one week or so~.

### Download the app 📱🔽 
* [Play Store (Android)](https://play.google.com/store/apps/details?id=app.jamku.client.m) or 
* [App Store (iOS / iPhone / iPad)](https://apps.apple.com/app/id1625118771)

Below are some of the screenshots of pages from the mobile app. Do let us know your suggestions as well.

### **1. Home page/ Dashboard**
![image](https://user-images.githubusercontent.com/99783627/157428709-86f7e920-67c2-4f67-bf27-86a29f8807e6.png)

### **2. Task List**
![image](https://user-images.githubusercontent.com/99783627/157430326-3baea40d-2bc4-4fdf-bafd-2869361cbb7a.png)

### **3. Client View**
It will show
1. Client Details - Name, PAN
2. Profile (which are marked as visible to client)
3. Shortcuts (aka Links) (Which are marked as visible to client)
4. Link to download files which are marked as visible to client
![image](https://user-images.githubusercontent.com/99783627/157431326-4f89ef99-c61b-4d31-a94b-eab179a539d4.png)

### **4. Task View**
It will show
1. Task Details - Status, Due Date, Completion Date
2. Stages (which are marked as visible to client)
3. Task Notes (Which are marked as visible to client)
4. Task Fields (Which are marked as visible to client)
5. Shortcuts (aka Links) (Which are marked as visible to client)
6. Link to download files which are marked as visible to client
![image](https://user-images.githubusercontent.com/99783627/157439973-931758a4-9cab-4bd4-bda8-8d17afe8a56b.png)

**Comment by ParinPujara** [comment]
It would be great if you add following features

1. DMS - All the documents like return copy etc. uploaded  by user at the time of completion of task, will save in seperate folder period wise task wise. E.g. one folder created for FY22 wherein all service folder will be created like GSTR 3B, GSTR1, INCOME TAX RETURN etc. So clients can easily download them without searching the said copy from task. 

2. Requested Service : There should be option for service request tab. Where clients can request for any service he want to avail. Like if clients needs any CA Certificate, he just request from his portal and we will see all the requested Services and can convert it into task. 
3. Outstanding dashboard: where client can find all his invoices raised and out of them which invoices are still pending for payment, he can view. 

Thank you🙏

**Reply by adarshmadrecha** [reply]
1. Will need to take this to drawing board.
2. Nice suggestion. Noted here - https://github.com/madrecha/jamku/issues/952
3. Yes Outstanding dashboard is coming in this app.

**Reply by ParinPujara** [reply]
Thank you for considering the suggestion.

**Comment by ParinPujara** [comment]
And it would be great if you provide the option of White label.

**Reply by adarshmadrecha** [reply]
I will be frank and straight in answering this.
As a principle we don't provide "White label". In cities where "Jamku" is popular, Offices feel proud that they are using Jamku. We want to create same experience for Clients. Where Clients would one day ask CA firms to start using Jamku, as their friend is using Jamku Client Portal.

**Reply by vatsalbhatia** [reply]
> And it would be great if you provide the option of White label.

I also feel this should be provided....

**Reply by ParinPujara** [reply]
Thank you Adarshbhai for your straight forward answer. 

May be you are creating brand identity for your company. We are also thinking in the same line. We don't have any issue with regard to using of 'Jamku'. But White Label will have some different impact on clients. For building our Brand Image in our market, it will be value addition tool for our clients. Nothing more than that.
